### PR TITLE
Tweak formatting & add notes on event retargeting.

### PR DESCRIPTION
**Pull request by arthurevans**
Also fixed a small typo in observers section.

### CODE REVIEW

**Comment by arthurevans** [comment]
@kayce PTAL. I did some formatting tweaks (and added some notes) to the events chapter after merging https://github.com/Polymer/docs/pull/1546, also a small typo fix in the observers section.

Mostly whitespace changes, could use a second pair of eyes for sanity checking.

File: 1.0/docs/devguide/events.md [hunk 1]
```diff
@@ -286,19 +267,19 @@ Example with `listeners`:
...
-
     </dom-module>
 
 
 ## Event retargeting {#retargeting}
 
 Shadow DOM has a feature called "event retargeting" which changes an event's
-target as it bubbles up, such that target is always in the receiving element's
-light DOM. Shady DOM does not do event retargeting, so events may behave differently
+target as it bubbles up, such that target is always in the same scope as the
```
Comment (kaycebasques) [inline 1.0/docs/devguide/events.md]: s/that target/that the target/

File: 1.0/docs/devguide/properties.md [hunk 2]
```diff
@@ -325,7 +325,7 @@ Example:
     });
 
 **Warning:** A single property observer shouldn't rely on any other properties,
-sub-properties, or paths because the observer can be called while these
```
Comment (kaycebasques) [inline 1.0/docs/devguide/properties.md]: Is my text editor adding a special character or something to the end of the lines?

File: 1.0/docs/devguide/events.md [hunk 3]
```diff
@@ -310,46 +291,62 @@ normalized event has the following properties:
...
 
     <script>
-      document.querySelector('event-retargeting').addEventListener('click', function(){
-        var normalizedEventObject = Polymer.dom(event);
-        console.info('rootTarget is:', normalizedEventObject.rootTarget); // logs #myButton
-        console.info('localTarget is:', normalizedEventObject.localTarget); // logs an instance of event-targeting that was originally queried
-        console.info('path is:', normalizedEventObject.path); // logs [#myButton, document-fragment, event-retargeting, body, html, document, Window]
+      document.querySelector('event-retargeting').addEventListener('click',
+          function(){
+        var normalizedEvent = Polymer.dom(event);
```
Comment (kaycebasques) [inline 1.0/docs/devguide/events.md]: Indentation

File: 1.0/docs/devguide/events.md [hunk 4]
```diff
@@ -286,19 +267,19 @@ Example with `listeners`:
...
 ## Event retargeting {#retargeting}
 
 Shadow DOM has a feature called "event retargeting" which changes an event's
-target as it bubbles up, such that target is always in the receiving element's
-light DOM. Shady DOM does not do event retargeting, so events may behave differently
+target as it bubbles up, such that target is always in the same scope as the
+receiving element. (For example, for a listener in the main document, the
+target is an element in the main document, not in a shadow tree.)
+
+Shady DOM does not do event retargeting, so events may behave differently
```
Comment (kaycebasques) [inline 1.0/docs/devguide/events.md]: When I read this I asked myself "why is there a difference between the two?" Could be helpful to add a line addressing that.

**Comment by kaycebasques** [comment]
A couple small items, otherwise LGTM

File: 1.0/docs/devguide/properties.md (hunk 2 above)
Comment (arthurevans) [inline 1.0/docs/devguide/properties.md]: Those are just extra spaces at the end of the line. I'm not sure if it's your text editor adding them, or just the fact that I'm mostly using Sublime Text nowadays, which removes the trailing spaces. (Or, mine is configured to remove trailing spaces.) Which editor are you using nowadays?

File: 1.0/docs/devguide/events.md (hunk 3 above)
Comment (arthurevans) [inline 1.0/docs/devguide/events.md]: This is intentional--when wrapping a long function invocation, the next argument's supposed to be either indented 4 spaces or aligned with the parens. I don't love the way this looks, but I also don't love the alternatives...

**Comment by arthurevans** [comment]
Addressed comments, merging.

**Pull request merged**